packages/abi-gen/test/generate_test/contracts/LibDummy.sol

```diff
@@ -0,0 +1,42 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.5;
+
+
+library LibDummy {
```

Comment:
Was this pulled from OpenZepplin?

Reply:
Nope!

Just took a quick look at their repos.  Is there something you have in mind that we could use here?

Reply:
Hm, can't seem to find the one I meant. Asked @abandeali1 about it.